templates/registry/go/dashboard-builder.go.j2

```diff
@@ -34,7 +34,7 @@ func main() {
 			Datasource(prometheusDataSourceRef).
 			WithTarget(
 				prometheus.NewDataqueryBuilder().
-					Expr("rate({{metric.metric_name}}[$__rate_interval])").
+					Expr("rate({{metric.metric_name | snake_case }}[$__rate_interval])").
```

Comment:
I can see why you did this. We use snake case in the instrumentation, so we need snake case here to match.

I'm not sure we need snake_case, though; Prometheus clients and PromQL already support UTF-8. Could we add a TODO to investigate this later?